wagon-providers/wagon-http/pom.xml

```diff
@@ -33,6 +33,19 @@ under the License.
     Wagon provider that gets and puts artifacts through HTTP(S) using Apache HttpClient-4.x.
   </description>
 
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+    <license>
```

Comment:
I am not sure about the official meaning of this tag.
Usually crawlers read this tag to classify the project.

This project is not MIT Licensed and it is not dual licensed.

Maybe it is better to add the 'comment' into the primary ALv2.0 license tag

Reply:
Maven defined the format, with following documentation https://maven.apache.org/ref/3.6.2/maven-model/maven.html#class_license

if you want to add a comment, just propose a content